### ISSUE TITLE
add muun: scheme to sendflow schemes bitcoin: & lightning:

### ISSUE THREAD

**Issue by Mvfaassen** [muun/apollo#3]
please add a muun: prefix additionally next to the current bitcoin: & lightning: schemes. Examples:

muun:lightning:{invoice}
&
muun:bitcoin:{address}?amount=

benefit over the current general deeplinks is that it will solely open the muun wallet, and won't have issues if there are more than 1 lightning: deeplink available on the device. This will overcome current issues IOS & Desktop has.

**Comment by esneider** [comment]
Thanks for reporting this!

As a temporary workaround, you can currently use `muun:external/{address}?amount={amount}&currency=btc`. We'll add support for `muun:{address}?amount={amount}` and `muun:{invoice}` in the next release. The new deep links will be fully compatible with the `bitcoin:` and `lightning:` URI format.

**Comment by Mvfaassen** [comment]
Thanks!

I tried our your method with an example address:  https://codepen.io/milocoinrequest/pen/WNNvOgz 

<br />
<a href='muun:external/3CMCRgEm8HVz3DrWaCCid3vAANE42jcEv9?amount=0.42&currency=btc
'>Muun general</a>
<br />

but it gives me an unexpected error, so not sure if it is active?

**Comment by slezica** [comment]
Hello @Mvfaassen! Santiago here, from Muun's development team.

I'm sorry you've had problems with this. The `muun:` scheme had never been used by 3rd parties before, and so we never paid it too much attention.

In the current release, the correct URL for Android looks like this:

```
muun:external/3CMCRgEm8HVz3DrWaCCid3vAANE42jcEv9?amount=0.3&currency=BTC
```

Note that the currency code `BTC` is in upper case.

As Dario said, in our next release we'll support links without the `external` part, both for Bitcoin addresses and Lightning Network invoices, making them compatible with the `bitcoin` and `lightning:` schemes.

Thanks for bringing this to our attention.

**Comment by Mvfaassen** [comment]
Hi @slezica , 

perfect, good to hear! I'll play with it and keep an eye out for the release. After that we will add you to the CoinRequest.io plaform for frictionless invoicing. 

Thanks for the quick fix!

**Comment by slezica** [comment]
@Mvfaassen hey!

I'm sorry for the late response. In our currently available application, the `muun:<address>` and `muun:<invoice>` URIs are supported.

The `muun:<address>` URI supports the `?amount=<btc>` parameter. The lightning URI does not, as amounts come encoded into the invoice.

We decided against supporting the `currency` parameter. It will always be BTC. To support it, we would've faced a hard problem:

The URI `<address>?amount=<n>&currency=<c>` would've been confusing. Is `n` in BTC, converted to `c`? Or is `n` measured in `c`? What does `amount=1&currency=USD` mean?

If we take the amount to be BTC and convert the value internally, we'll most likely be using a different exchange rate than the origin of the link. Even if it's off by a very small margin it will fail to add up.

If we take the amount to be USD, we'll break compatibility with `bitcoin:` URIs.

Bottom line, you can use `muun:` to open both addresses (with the optional `amount` parameter) and lightning invoices in the Muun application, but the `currency` parameter is not available.
